### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by spaette**
PSReadLine/Changes.txt
PSReadLine/Cmdlets.cs
PSReadLine/Completion.cs
PSReadLine/DisplayBlockBase.cs
PSReadLine/Keys.cs
PSReadLine/Movement.cs
<s>PSReadLine/Movement.vi.cs</s>
PSReadLine/PSReadLineResources.Designer.cs
PSReadLine/PSReadLineResources.resx
PSReadLine/Prediction.Views.cs
PSReadLine/ReadLine.cs
PSReadLine/Render.Helper.cs
PSReadLine/Render.cs
PSReadLine/YankPaste.vi.cs
build.ps1
test/CompletionTest.cs
test/MockConsole.cs
test/RenderTest.cs
tools/CheckHelp.ps1
tools/releaseTools.psm1

fixes primarily comments

the exception being code in Movement.cs

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/PowerShell/PSReadLine/pull/3547)

### CODE REVIEW

**Comment by spaette** [comment]
@microsoft-github-policy-service agree [company="{your company}"]

**Comment by spaette** [comment]
@microsoft-github-policy-service agree

File: PSReadLine/Movement.vi.cs [hunk 1]
```diff
@@ -152,7 +152,7 @@ public static void ViEndOfPreviousGlob(ConsoleKeyInfo? key = null, object arg =
         /// <summary>
         /// Returns 0 if the cursor is allowed to go past the last character in the line, -1 otherwise.
         /// </summary>
-        /// <seealso cref="ForwardChar"/>
+        /// <see also cref="ForwardChar"/>
```
Comment (daxian-dbw) [inline PSReadLine/Movement.vi.cs]: `<seealso>` is a legit XML doc element. See https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#seealso
```suggestion
        /// <seealso cref="ForwardChar"/>
```

**Review by daxian-dbw — APPROVED** [review]
LGTM with just one correction

**Review by daxian-dbw — APPROVED** [review]

**Comment by daxian-dbw** [comment]
@spaette Thanks for the contribution!

**Comment by spaette** [comment]
@daxian-dbw 

_off topic_

Concomitant with typo fixes in PowerShell/PowerShell would removal of a word from .spelling be authorised?

```
$ ed -s PowerShell/CHANGELOG/7.1.md <<<'696p'
<li>Add missing assessibility modifiers (#12820) (Thanks @xtqqczze!)</li>
$ 
```

The merged PR above had a link to a page where `roslynator_accessibility_modifiers` was mentioned.

```
Add missing assessibility modifiers
Thu, 28 May 2020
```

The spelling in the commit message was of course incorrect and that spelling found its way into .spelling.

```
$ ed -s PowerShell/.spelling <<<'73p'
assessibility
$ 
```

**Pull request merged**